habitat/tests/test_versions.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
```

Comment:
The only reason I went this `minitest` route was to align with how inspec performs their verification tests while allowing for a growth path should we end up wanting to add more complicated tests.

Please let me know if this is just silly and whether I should simplify this by tossing the multiple calls to `hab pkg exec` into `scripts/ci/verify-plan.sh`